package.json

```diff
@@ -55,6 +55,7 @@
     "xsd-schema-validator": "^0.7.0"
   },
   "dependencies": {
+    "bpmn-in-color-moddle": "^0.1.0",
```

Comment:
We should make it a DEV dependency, as it is [inlined in the bundle](https://github.com/bpmn-io/bpmn-moddle/blob/master/rollup.config.js#L56).

Alternative: Make it an explicit package dependency. I'd be fine with inlining.